_layouts/home.html

```diff
@@ -9,137 +9,157 @@
-                            {% endif %} -->
-                          </span></p>
-                        </div>
-                      </a>
-                    {% endif %}
-                  {% endfor %}
-                {% endfor %}
-                
+          <img id="slide-5" src="{{ project['Feature Image'] }}" alt="{{ project.title }}"/>
+          <div class="overlay-alt">
+            <div class="overlay-text">
+              <h3 class="home-carrousel-projects" style="background-color: white; padding: 2%;">EVENTS: <a style="color: black;" href="{{ project.url }}">{{ project['Summary Text'] }}</a></h3>
+              {% endif %}
+              {% endfor %}
+          {% endfor %}
+            </div>  
+          </div>
+        </div>
+      </div>
 
-              </div>
-            </div>
+      <div class="slider-nav">
+        <a href="#slide-1" data-slide="0"></a>
+        <a href="#slide-2" data-slide="1"></a>
+        <a href="#slide-3" data-slide="2"></a>
+        <a href="#slide-4" data-slide="3"></a>
+        <a href="#slide-5" data-slide="4"></a>
+      </div>
 
-            <div class="container">
-              <header class="home-header">
-                <!-- <h1>{{ content }}</h1>
-                <div class="home-cta">
-                  <a href="/what-we-do" class="btn btn-primary btn-lg btn-chevron btn-block">
-                    Learn about what we do
-                  </a>
-                  <a href="https://tasks.hotosm.org/explore" class="btn btn-info btn-lg btn-chevron btn-block class start-mapping-btn ttu font-headline">
-                    Start mapping
-                  </a>
-                </div> -->
-              </header>
-            </div>
+      <!-- Arrows for navigation -->
+      <button class="arrow left">&lt;</button>
+      <button class="arrow right">&gt;</button>
+    </div>
+  </section>
+
+  <script src="/js/carrousel-homepage.js"></script>
   
-            <div class="home-highlights-controls">
-              <div class="home-controls">
-                <div class="control" id="control-left">Left</div>
-                <div class="control" id="control-right">Right</div>
-            </div>
 
-          </div>
+
+
+        
+
+
+
 
           <div class="container home-stats">
             <div class="home-stat">
-              <!-- <center><p class="loader"></p></center> -->
-              <!-- <p class="stat-number" id="Community-Mappers"></p> -->
-              <p class="stat-number">468K</p>
+              <p class="stat-number">491K</p>
               <p class="stat-title">Total Mappers<sup>&#9432;</sup>
-                <span class="tooltiptext">Total number of Mappers using <a href="https://tasks.hotosm.org/" target="_blank">HOT Tasking Manager</a>  - As of Dec. 2023</span>
+                <span class="tooltiptext">Total number of Mappers using <a href="https://tasks.hotosm.org/" target="_blank">HOT Tasking Manager</a>  - as of May 2024</span>
```

Comment:
We still manually update this ?

Reply:
For future : ohsomenow provides and API endpoint from which you can query and display the latest numbers